stages/org.osbuild.ostree.config

```diff
@@ -44,6 +44,10 @@ SCHEMA = """
             "type": "boolean"
           }
         }
+      },
+      "bls_append_except_default": {
```

Comment:
i'm guessing `-` aren't accepted in the names in the schema, otherwise we would use `bls-append-except-default`?

Reply:
Using `-` seems to be accepted in the yaml, changed underscores to dashes.